tasks/jenkins.rake

```diff
@@ -3,17 +3,24 @@ require 'English'
 begin
   require "ci/reporter/rake/test_unit"
   namespace :jenkins do
+    desc 'Sets up pre-CI environment for testing'
```

Comment:
This doesn't just set it up: it requires both the setup and then runs the tests.